services/director-v2/setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.2.0
+current_version = 2.3.0
```

Comment:
I know this is not your change. But maybe just read in the version from `VERSION` (or the other way around). At least I guess it should not be duplicated.

Reply:
@bisgaard-itis `bumpversion` is a special tool that synchronizes version across the package. Let me show you offline :-)